docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -1,8 +1,82 @@
-# Verifying Cluster Health
+# Cluster Health and Troubleshooting
 
-Following is a set of tests that determine the Run:ai cluster health:
+## Cluster is Disconnected
 
-## Verify that data is sent to the cloud
+When a cluster is in the status “Disconnected”, this means that no communication from the Run:ai cluster services reaches the Run:ai Control Plane. 
+
+This may reflect a networking issue from or to your Kubernetes cluster regardless of Run:ai components. In some cases, it may indicate an issue with one or more Run:ai services that communicate with the Control Plane, which are:
+* Cluster sync (`cluster-sync`)
+* Agent (`runai-agent`)
+* Asset sync (`asset-sync`)
+
+### Troubleshooting actions:
+
+* First, check that the Run:ai services that communicate with the Control Plane are up and running.
+
+  Run:
+
+  `kubectl get pods -n runai | grep -E 'runai-agent|cluster-sync|assets-sync'`
+
+* Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is an essential first step to identify any error messages or connection issues.
+
+  Run the following command on each one of the services:
+
+  ```
+  kubectl logs runai-agent -n runai
+  kubectl logs cluster-sync -n runai
+  kubectl logs assets-sync -n runai
+  ```
+
+* Check the network connection from the runai namespace in your cluster to the Control Plane. You can do that by running a connectivity check pod.
+
+  Create a pod within the runai namespace. This pod can be a simple container with basic network troubleshooting tools, such as curl or get. Use the following command to determine if the pod can establish connections to the necessary Control Plane endpoints:
+
+  `kubectl run control-plane-connectivity-check -n runai --image=wbitt/network-multitool --command -- /bin/sh -c 'curl -sSf <control-plane-endpoint> > /dev/null && echo "Connection Successful" || echo "Failed connecting to the Control Plane"'`
+
+  Replace `control-plane-endpoint` with the URL of the Control Plane in your environment.
+
+* Check potential network issues. You can use the following guidelines:
```

Comment:
@jasonnovichRunAI the reason I wrote "you can" is because these are not necessarily all the steps that can be done... I want to emphasize that these guidelines should help you but you may have other options to check the network in your env. How can we do that?